.kitchen.yml

```diff
@@ -1,35 +1,40 @@
 ---
+<% chef_versions = %w( 13 14 ) %>
+<% platforms = %w( centos-6 centos-7 ) %>
+
 driver:
   name: vagrant
   linked_clone: true
-  require_chef_omnibus: 12
 
 provisioner:
   name: chef_zero
+  log_level: <%= ENV['TK_CHEF_LOG_LEVEL'] || 'info' %>
 
 verifier:
   name: inspec
   inspec_tests:
     - test/inspec
 
 platforms:
-  - name: centos-6.8
-  - name: centos-7.3
-
-suites:
-  - name: config_file
-    run_list:
-      - recipe[knotx::_install]
+<% platforms.each do |p| %>
+<%   chef_versions.each do |chef_version| %>
+  - name: <%= p %>-chef-<%= chef_version %>
+    driver_config:
+      box: bento/<%= p %>
+      require_chef_omnibus: <%= chef_version %>
     attributes:
       knotx:
-        debug_enabled: true
-  - name: config_git
+        url: 'https://oss.sonatype.org/content/groups/public/io/knotx/knotx-stack-manager/1.3.0-SNAPSHOT/knotx-stack-manager-1.3.0-20180417.122358-11.zip'
```

Comment:
You should have the version to be configurable here

Reply:
Btw, there are newest snapshot available - so you can use it for your tests:
`https://oss.sonatype.org/content/repositories/snapshots/io/knotx/knotx-stack-manager/1.3.0-SNAPSHOT/knotx-stack-manager-1.3.0-20180425.065432-12.zip`

Reply:
Version is still configureable. You can either do 

```
knotx_instance 'first' do
  # URL is calculated from ['knotx']['release_url'] attribute. It won't work for SNAPSHOTS as
  # "1.3.0-SNAPSHOT" and "1.3.0-20180425.065432-12" don't match. In such case solution
  # with "source" is preferred. Of course it is possible to introduce yet another resource 
  # property to control this on a more granular level.
  #
  # Example: $URL_BASE/$VER/knotx-stack-manager-$VER.zip
  version '1.3.0'
end
```

or 

```
knotx_instance 'second` do
  source 'https://whatever.the/url/is/to/knotx.zip'
end
```

The only reason this attribute exist in `.kitchen.yml` is to have a single place for updates when `1.3.0` is not out yet.